Source/CarthageKit/Xcode.swift

```diff
@@ -950,6 +950,25 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 	}
 
 	return BuildSettings.SDKsForScheme(scheme, inProject: project)
+		.flatMap(.Concat) { sdk -> SignalProducer<(SDK, BuildSettings), CarthageError> in
+			var argsForLoading = buildArgs
+			argsForLoading.sdk = sdk
+
+			return SignalProducer(value: sdk)
+				.zipWith(BuildSettings.loadWithArguments(argsForLoading))
```

Comment:
Actually, I think you could move the `filter` inside here and get rid of the tuple  and the second `flatMap` altogether:

```swift
return BuildSettings
    .loadWithArguments(argsForLoading)
    .filter { settings in 
        return settings.bitcodeEnabled.value || ![.tvOS, .watchOS].contains(sdk)
    }
    .map { _ in sdk }
```